packages/cli/src/ceramic-daemon.ts

```diff
@@ -95,12 +95,15 @@ class CeramicDaemon {
       })
     }
 
+    // next is required in function signature
     app.use((err: Error, req: Request, res: Response, next: NextFunction): void => {
-      const requestStart = Date.now()
-      const httpLog = this._buildHttpLog(requestStart, req, res)
-      const logString = JSON.stringify(httpLog)
-      this.logger.error(logString)
-      next(err)
+      this.logger.error(err)
+      if (res.statusCode < 300) { // 2xx indicates error has not yet been handled
+        res.status(500)
```

Comment:
I guess I don't really understand enough about how HTTP status codes are used.  Why wouldn't we just use `res.statusCode` here?  Aren't we losing information by not using the statusCode and just always using 500?

Reply:
I assume this handler will only get called if there is an error elsewhere. Then if the status code has already been set to 200 we should change it to 500 in order to reflect that there was an error.

Reply:
Yes exactly @oed. Also @stbrody this is where the "Ceramic failed a healthcheck" will get logged